packages/teleterm/src/ui/DocumentGateway/Mongo/useMongo.ts

```diff
@@ -16,15 +16,7 @@ limitations under the License.
 
 import { Gateway } from 'teleterm/ui/services/clusters';
 
-export type MongoProps = {
-  title: string;
-  disconnect(): void;
-  gateway: Gateway;
-};
-
-export function useMongo(props: MongoProps) {
-  const { gateway, disconnect, title } = props;
-
+export function useMongo(gateway: Gateway) {
```

Comment:
Technically these `useMongo` type functions no longer need to live in protocol-specific folder, but we're going to remove them anyway later on. So I just kept them where they are.